api/v1beta1/cloudstackcluster_webhook_test.go

```diff
@@ -14,198 +14,85 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1beta1
+package v1beta1_test
 
 import (
 	"context"
 
+	infrav1 "github.com/aws/cluster-api-provider-cloudstack/api/v1beta1"
+	"github.com/aws/cluster-api-provider-cloudstack/test/dummies"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
 var _ = Describe("CloudStackCluster webhooks", func() {
-	const (
```

Comment:
Should we be concerned about putting all the test variables in one file?  On one hand, it helps to prevent duplicated names in the same package.  On the other hand, it could turn into a bloated dumping ground for variables.

Reply:
I think maybe we should be concerned about it becoming a dumping ground, but I think that's about quality -- not whether we should do it or not.

If it does grow much more, we'll want to restructure it. Possibly using sub modules or whatnot. Either way, using test/dummies/vars.go has dropped hundreds of lines from tests that only made testing harder to grok.

We just need to be careful about adding every little thing to the dummies file vs just making small tweaks to existing dummies in the tests themselves.

I'm thinking of them as a launchpad.